pubsub/iam.js

```diff
@@ -13,69 +13,88 @@
 
 'use strict';
 
-var async = require('async');
-var subscriptionSample = require('./subscription');
-var createTopicExample = subscriptionSample.createTopicExample;
-var deleteTopicExample = subscriptionSample.deleteTopicExample;
-var subscribeExample = subscriptionSample.subscribeExample;
-var deleteSubscriptionExample = subscriptionSample.deleteSubscriptionExample;
-var pubsub = subscriptionSample.pubsub;
+// [START auth]
+// By default, gcloud will authenticate using the service account file specified
+// by the GOOGLE_APPLICATION_CREDENTIALS environment variable and use the
+// project specified by the GCLOUD_PROJECT environment variable. See
+// https://googlecloudplatform.github.io/gcloud-node/#/docs/guides/authentication
+var gcloud = require('gcloud');
+
+// Instantiate a pubsub client
+var pubsub = gcloud.pubsub();
+// [END auth]
 
 // [START get_topic_policy]
 /**
- * @param {string} topicName Name of the topic whose policy is to be retrieved.
- * @param {Function} callback Callback function.
+ * Retrieve a topic's IAM policy.
+ *
+ * @param {string} topicName The name of the topic.
+ * @param {function} callback The callback function.
  */
-function getTopicPolicyExample (topicName, callback) {
+function getTopicPolicy (topicName, callback) {
+  if (!topicName) {
+    return callback(new Error('"topicName" is required!'));
+  }
+
   // Grab a reference to an existing topic
   var topic = pubsub.topic(topicName);
 
   // Retrieve the IAM policy for the topic
-  topic.iam.getPolicy(function (err, policy, apiResponse) {
+  topic.iam.getPolicy(function (err, policy) {
     if (err) {
       return callback(err);
     }
 
-    // Received the policy
-    console.log(policy); // { etag: 'ACAB' }
-    callback(null, policy, apiResponse);
+    console.log('Got policy:', policy);
```

Comment:
Should this be "got **topic** policy" to differentiate from the other functions?